Source/Server/MqttService.cpp

```diff
@@ -165,12 +137,12 @@ namespace MQTT {
 			return nullptr;
 		}
 
-
-		void MqttService::OnClientDisconnect(const Client& client, const Protocol::DisconnectPackage& package)
+		std::string MqttService::GenerateUniqueId()
 		{
-			//TODO: Remove will message when storage of it is implemented.
-			DisconnectClientState(client);
+			// TODO: Create unique id
+			return "1";
```

Comment:
Use the generate uid from client utility

Reply:
Resolved by deleting method. Not used anymore